modules/network-modifying-operator-install-config.adoc

```diff
@@ -0,0 +1,144 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-network-customizations.adoc
+
+[id="modifying-nwoperator-config-startup-{context}"]
+= Modifying advanced network configuration parameters
+
+You can modify the advanced network configuration parameters only before you
+install the cluster.
+
+[NOTE]
+====
+You must use the installer program to create an `install-config.yaml` file
+before continuing.
+====
+
+.Prerequisites
+
+* The `install-config.yaml` configuration file for the cluster you are creating.
+
+.Procedure
+
+. Use the following command to create manifests:
++
+----
+$ openshift-install create manifests --dir=<directory> <1>
+----
+<1> Specify the directory name to store the files that the installation program
+creates.
++
+The installer creates a `manifests/` directory that contains the
+`cluster-network-02-config.yml` file describing a `Network.config.openshift.io`
+Custom Resource (CR):
++
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: Network
+metadata:
+  creationTimestamp: null
+  name: cluster
+spec:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  networkType: OpenShiftSDN
+  serviceNetwork:
+  - 172.30.0.0/16
+status: {}
+----
++
+The Network Operator takes its configuration from the
+`cluster-network-02-config.yml` file when the cluster is created. Both the
+`Network.config.openshift.io` CR and the `Network.operator.openshift.io` CR must
```

Comment:
This isn't the case - the files can be named anything.

Reply:
But `cluster-network-02-config.yml` is already created by the installer.

However, AFAICT when testing, this paragraph is explicitly wrong: the installer assumes that each file contains only a single object, and so the cluster config and operator config need to be in separate files

Reply:
You need to put the `config.openshift.io` object and the `operator.openshift.io` object in separate files, rather than concatenating them together into a single file like it says here.

Reply:
Are there any limitations on the filenames?

Reply:
The `operator.openshift.io` object has to be in a file that comes alphabetically after `cluster-network-01-crd.yml`. Other than that, no. `cluster-network-03-config.yml` would be a fine name for it.